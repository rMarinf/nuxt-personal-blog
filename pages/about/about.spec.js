import { shallowMount } from '@vue/test-utils'
import About from './index.vue'

test('About', () => {
  // render the component
  const wrapper = shallowMount(About)

  expect(wrapper.element).toMatchSnapshot()
})
