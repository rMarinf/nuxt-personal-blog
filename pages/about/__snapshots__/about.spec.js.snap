// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About 1`] = `
<div
  id="about-me"
>
  
  ABOUT

</div>
`;
